src/edgepi/tc/edgepi_tc.py

```diff
@@ -254,7 +323,7 @@ def __get_tc_type(self):
         cr1 = Bits(uint=self.__read_register(TCAddresses.CR1_R.value)[1], length=8)
         tc_bits = cr1[-4:].uint
         for enum in TCType:
-            if enum.value.op_code == tc_bits:
+            if not isinstance(enum.value, int) and enum.value.op_code == tc_bits:
                 return enum
```

Comment:
Not a big deal but `enum` is also the name of the library you imported to get the `Enum` class. I'm not sure if it's good naming practice to use `enum` again here.

Reply:
good point and fixed aef1556b20336d5d0c033db5c414ed15cc7184e0